buildSrc/src/main/groovy/com/google/edwmigration/dumper/build/licensereport/OverridesUtil.groovy

```diff
@@ -0,0 +1,794 @@
+package com.google.edwmigration.dumper.build.licensereport
+
+class OverridesUtil {
+
+    static Map<String, Map<String, String>> projectOverrides() {
```

Comment:
I would suggest to return an `immutable` map.

Reply:
It's safe to modify this map, because the method creates a new copy each time. The api design with a Map is not ideal, but I'm not sure that making it immutable will solve the problems.